EosPayload/drivers/cutdown_driver.py

```diff
@@ -0,0 +1,97 @@
+from queue import Queue
```

Comment:
This driver needs updating to be compatible with the new driver handling, not sure if that's a problem for another issue